services/website/content/en/stake-pools/7e45a7e6ab3afcf99120e97aedf84e706e43d829ddc610ad667a85a3.md

```diff
@@ -5,9 +5,27 @@ memberSince: 2021-06-15
 identities: 
     - id: pgwad
       role: spo
+nodes:
+  - role: producer
+    deviceId: raspberry-pi-4-8gb
+    storage: WD_Black NVMe SSD 512GB
+  - role: relay 
+    deviceId: raspberry-pi-4-8gb
+    storage: WD_Black NVMe SSD 512GB
+  - role: relay 
+    deviceId: raspberry-pi-4-8gb
+    storage: WD_Black NVMe SSD 512GB
+  - role: relay
+    name: Backup1 
+    deviceId: raspberry-pi-4-8gb
+    storage: WD_Black NVMe SSD 512GB
+  - role: relay
```

Comment:
don't include the backups

Reply:
backups are ok, just add `isBackup: true`

Reply:
but why? I don't find any value in knowing you have backups

Reply:
Well, if Sub says to do it, then do it :)

Reply:
it's alright, we can always scrub some data off. harder than to collect it.